hammer/reporting-remediation/reporting/create_cloudtrail_tickets.py

```diff
@@ -56,7 +56,23 @@ def create_tickets_cloud_trail_logging(self):
                 region = issue.issue_id
                 # issue has been already reported
                 if issue.timestamps.reported is not None:
-                    if issue.status in [IssueStatus.Resolved, IssueStatus.Whitelisted]:
+                    if issue.status in [IssueStatus.Tempwhitelist]:
+                        logging.debug(f"CloudTrail logging issue with '{region}' "
+                                      f"is added to temporary whitelist. ")
+
+                        comment = (f"CloudTrail logging issue with '{region}' "
+                                   f"in '{account_name} / {account_id}' account is added to temporary whitelist.")
+                        jira.update_issue(
```

Comment:
Will this message be posted to jira for the same issue multiple times (on each reporting phase)?

Reply:
I don't think we should add this message more than once, at least we shouldn't add it if nothing is changed.